apps/coscrad-frontend/src/components/term-of-the-day/use-date.tsx

```diff
@@ -0,0 +1,7 @@
+export function useDate() {
+    const currentDay = new Date().getDate();
+
+    console.log('TODAYS DAY:', currentDay);
+
```

Comment:
enum Month{
   January='January';
   February='February';
// ...
    December='December';
}

type MonthAndDay = {
month: Month;
day: string; // or maybe `${number}` or painfully '1' | '2' | .... | '31'
}

Parse the raw data and determine the month and date.

Reply:
We found a solution to this when pairing.